.github/scripts/validateImmutableActionRefs.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+############################################
+#    Check for unsafe action references    #
+############################################
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+readonly SCRIPT_DIR
+GITHUB_DIR="$(dirname "$SCRIPT_DIR")"
+readonly GITHUB_DIR
+
+source "$SCRIPT_DIR/shellUtils.sh"
```

Comment:
Similar to above, you can simplify:

```suggestion
GITHUB_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")/.." &>/dev/null && pwd)"
readonly GITHUB_DIR

source "$GITHUB_DIR/scripts/shellUtils.sh"
```

Reply:
done